yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -224,7 +247,17 @@ export class Sequencer {
     const chainTipArchive =
       chainTip == undefined ? new Fr(GENESIS_ARCHIVE_ROOT).toBuffer() : chainTip?.archive.root.toBuffer();
 
-    let slot: bigint;
+    const currentL1Timestamp = await this.publisher.getL1Timestamp();
+    let slot = getSlotAtTimestamp(currentL1Timestamp, this.publisher.l1ContractConstants);
```

Comment:
this `slot` and the one on line 262 that shadows it aren't the same thing which could be confusing. Maybe should instead just keep the original `let slot: bigint;` and separately
```
    this.loggingContext = {
      currentL1Timestamp,
      slot: getSlotAtTimestamp
      newBlockNumber,
      chainTipArchive,
    };
```